### PR TITLE
[playground] Use non-zero gas fee for transaction

### PR DESCRIPTION
**Pull request by cf19drofxots**
### Description

This estimates how much gas to use for the deposit transaction.

- [ ] Deploy preview is functional

### CODE REVIEW

File: packages/typescript-typings/types/ethers/index.d.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ declare class Web3Provider {
   getBalance(address: string): Promise<BigNumber>;
   on(event: string, callback: (...args) => void): void;
   removeAllListeners(event: string): void;
+  estimateGas(tx): Promise<any>;
```
Comment (snario) [inline packages/typescript-typings/types/ethers/index.d.ts]: I'd just remove this entire folder altogether, its outdated

File: packages/playground/src/components/account/account-deposit/account-deposit.tsx [hunk 2]
```diff
@@ -30,9 +31,14 @@ export class AccountDeposit {
...
-      await this.signer.sendTransaction({
+      const tx = {
         to: this.user.multisigAddress,
         value: this.amountDeposited
+      };
+      const gasEstimate = await this.provider.estimateGas(tx);
+      await this.signer.sendTransaction({
+        ...tx,
+        gasPrice: gasEstimate
       });
```
Comment (snario) [inline packages/playground/src/components/account/account-deposit/account-deposit.tsx]: Probably will look cleaner to just inline all of these vars

File: packages/typescript-typings/types/ethers/index.d.ts (hunk 1 above)
Comment (joelalejandro) [reply packages/typescript-typings/types/ethers/index.d.ts]: @snario It's not relevant at this moment, and also until Ethers support ES modules (ethers-io/ethers.js#114) we can't import it properly into Stencil.

Comment (cf19drofxots) [reply packages/typescript-typings/types/ethers/index.d.ts]: Well apparently it's still being used...

cc @joelalejandro

File: packages/playground/src/components/account/account-deposit/account-deposit.tsx (hunk 2 above)
Comment (joelalejandro) [reply packages/playground/src/components/account/account-deposit/account-deposit.tsx]: @snario Doesn't seem like a blocker.

**Pull request merged**